src/components/Calculator.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+/* eslint-disable */
+
```

Comment:
- [ ] Kindly don't disable linters, if errors come up it's better to fix them. Kindly remove `eslint-disable`, line two.